### PR TITLE
Start to test against PHP 7.3

### PR DESCRIPTION
**Pull request by carusogabriel**
Travis CI has already provided the PHP 7.3 images :)

### CODE REVIEW

**Comment by jgrossi** [comment]
thanks for the PR! it seems we need some changes on Travis CI config file to make it working 😎

**Comment by carusogabriel** [comment]
@jgrossi Sure, gonna work on this patch a little more

File: .travis.yml [hunk 1]
```diff
@@ -3,12 +3,13 @@ language: php
...
   - 7.1
   - 7.2
+  - 7.3
 
 dist: trusty
 sudo: required
 
 before_script:
-  - phpenv config-rm xdebug.ini
+  - if [ $TRAVIS_PHP_VERSION != "7.3" ]; then phpenv config-rm xdebug.ini; fi
```
Comment (carusogabriel) [inline .travis.yml]: @jgrossi If you have a better solution, please, update this patch. Otherwise, should be RTM :blush:

Comment (jgrossi) [reply .travis.yml]: actually, I added the xdebug back to enable code coverage. I changed some things but it seems to be working fine. thanks for sharing @carusogabriel 💪🔥

**Pull request merged**